README.md

```diff
@@ -44,9 +44,11 @@ chmod +x run.sh
 3. Run the code:
 
 ```
-./run.sh --listener_url=<listener_url> --token=<token> --gcp_region=<gcp_region> --function_name_prefix=<function_name> --metric_types=<metric_types>
+./run.sh --listener_url=<listener_url> --token=<token> --gcp_region=<gcp_region> --function_name=<function_name> --telemetry_list=<telemetry_list>
 ```
 
+<b>When you run this script, you should choose the project ID where you need to run the integration.</b>
```

Comment:
```suggestion
**When you run this script, you should choose the project ID where you need to run the integration.**
```
Those tags won't work in our docs, please replace them with the markdown equivalent (`**`).